src/tests/Equinor.ProCoSys.IPO.Query.Tests/GetInvitationsByCommPkgNo/GetInvitationsByCommPkgNoQueryHandlerTests.cs

```diff
@@ -0,0 +1,221 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Equinor.ProCoSys.IPO.Infrastructure;
+using Equinor.ProCoSys.IPO.Query.GetInvitationsByCommPkgNo;
+using Equinor.ProCoSys.IPO.Test.Common;
+using Fusion.Integration.Http.Models;
+using Fusion.Integration.Meeting;
+using Fusion.Integration.Meeting.Http.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using ServiceResult;
+
+namespace Equinor.ProCoSys.IPO.Query.Tests.GetInvitationsByCommPkgNo
+{
+    [TestClass]
+    public class GetInvitationsByCommPkgNoQueryHandlerTests : ReadOnlyTestsBase
+    {
+        private Invitation _dpInvitation;
+        private Invitation _mdpInvitation;
+        private int _dpInvitationId;
+        private int _mdpInvitationId;
+        private const string _commPkgNo = "CommPkgNo";
+        private const string _projectName = "Project1";
+
+        private Mock<IFusionMeetingClient> _meetingClientMock;
+
+        protected override void SetupNewDatabase(DbContextOptions<IPOContext> dbContextOptions)
+        {
+            using (var context = new IPOContext(dbContextOptions, _plantProvider, _eventDispatcher, _currentUserProvider))
+            {
+                var MeetingId = new Guid("11111111-2222-2222-2222-333333333333");
+                var PersonAzureOid = new Guid("44444444-5555-5555-5555-666666666666");
+                const string Description = "Description";
+
+                var FunctionalRoleParticipant = new Participant(
+                    TestPlant,
+                    Organization.Contractor,
+                    IpoParticipantType.FunctionalRole,
+                    "FR1",
+                    null,
+                    null,
+                    "FR1@email.com",
+                    null,
+                    0);
+
+                var PersonParticipant = new Participant(
+                    TestPlant,
+                    Organization.ConstructionCompany,
+                    IpoParticipantType.Person,
+                    null,
+                    "FirstName",
+                    "LastName",
+                    "P1@email.com",
+                    PersonAzureOid,
+                    1);
+
+                var CommPkg = new CommPkg(
+                    TestPlant,
+                    _projectName,
+                    _commPkgNo,
+                    Description,
+                    "OK");
+
+                var McPkg = new McPkg(
```

Comment:
Maybe add a test where we have two mc pkgs on the same IPO, and that we only get one IPO from endpoint?